second_2024_backend/427-microwave/README.md

```
@@ -0,0 +1,43 @@
+1 с
+
+Ограничение памяти
+
+256 МБ
+
+## Пример 1
+
+Ввод  | Вывод
+------|------
+6     | 5
+2 3 4 |
```

markdownlint-cli2 (0.17.2)

Table pipe style
Expected: no_leading_or_trailing; Actual: trailing_only; Unexpected trailing pipe

(MD055, table-pipe-style)